ci/collect-repo-info.py

```diff
@@ -151,7 +151,7 @@ def included_plugin_dir_conv(v):
     # Attempt to collect info from git first and alternatively
     # fall back to a repo-info.json file within what is assumed
     # to be a tarball.
-    if (zeek_dir / ".git").is_dir():
+    if (zeek_dir / ".git").exists():
```

Comment:
This fixes the issue I saw.

There is a similar check below, maybe it would make sense to put this into some function `is_git_checkout`, https://github.com/zeek/zeek/blob/252cd33283eac6bbe0721cc158faf302c5468868/ci/collect-repo-info.py#L114

That could also allow us change this the more verbose "Git API" instead of relying on implementation details (which could be easily fooled), e.g., check `git rev-parse --is-inside-work-tree` instead of simply checking for existence of `.git`.

Reply:
Applied your suggestion.